hca_util/__main__.py

```diff
@@ -32,45 +34,46 @@ def parse_args(args):
     parser_config.add_argument('ACCESS_KEY', help='AWS Access Key ID')
     parser_config.add_argument('SECRET_KEY', help='AWS Secret Access Key')
 
-    parser_create = cmd_parser.add_parser('create', help='create an upload directory (authorised user only)')
-    parser_create.add_argument('-n', metavar='name', help='optional project name for new directory', type=valid_project_name)
+    parser_create = cmd_parser.add_parser('create', help='create an upload area (authorised users only)')
+    parser_create.add_argument('name', help='name for the new area', type=valid_project_name)
```

Comment:
follow convention i used - caps letters for mandatory argument. change 'name' to 'NAME'